src/core/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,934 @@
+   */
+  act(() => {
+    fireEvent.pointerDown(slideContainer, {
+      pointerId: 5,
+      buttons: 1,
+      clientX: 310,
+    });
+  });
+
+  const thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.getAttribute('aria-valuenow')).toEqual('0.25');
+});
+
+it('should move closest thumb when pointer down on rail', () => {
+  const wrapper = render(
+    <Slider
+      values={[10, 80]}
+      tickLabels={<span className='custom-tick-mark'>Custom</span>}
+    />,
+  );
+
+  const { container } = wrapper;
+  const slideContainer = container.querySelector(
+    '.iui-slider-container',
+  ) as HTMLDivElement;
+  assertBaseElement(container);
+
+  /* fire a pointer down event 70% down the the slider */
+  act(() => {
+    fireEvent.pointerDown(slideContainer, {
+      pointerId: 5,
+      buttons: 1,
+      clientX: 710,
+    });
+  });
+
+  const thumbs = container.querySelectorAll('.iui-slider-thumb');
+  expect(thumbs.length).toBe(2);
+
+  expect(thumbs[0].getAttribute('aria-valuenow')).toEqual('10');
+  expect(thumbs[1].getAttribute('aria-valuenow')).toEqual('70');
+});
+
+it('should activate thumb on pointerDown and move to closest step on move', () => {
+  const handleOnUpdate = jest.fn();
+  const wrapper = render(
+    <Slider
+      min={0}
+      max={100}
+      values={[20, 80]}
+      step={1}
+      tickLabels={<span className='custom-tick-mark'>Custom</span>}
+      onUpdate={handleOnUpdate}
+    />,
+  );
+
+  const { container } = wrapper;
+  assertBaseElement(container);
+  const slideContainer = container.querySelector(
+    '.iui-slider-container',
+  ) as HTMLDivElement;
+  const thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.classList.contains('iui-active')).toBeFalsy();
+
+  act(() => {
+    fireEvent.pointerDown(thumb, {
+      pointerId: 5,
+      buttons: 1,
+      clientX: 210,
+    });
+  });
+  expect(thumb.classList.contains('iui-active'));
+
+  // moving to same location should not trigger update
+  act(() => {
+    fireEvent.pointerMove(slideContainer, {
+      pointerId: 5,
+      buttons: 1,
+      clientX: 210,
+    });
+  });
+
+  /* move thumb to 40 value on slider */
+  act(() => {
+    fireEvent.pointerMove(slideContainer, {
+      pointerId: 5,
+      buttons: 1,
+      clientX: 410,
+    });
+  });
+  expect(handleOnUpdate).toHaveBeenCalledTimes(1);
+
+  act(() => {
+    fireEvent.pointerUp(slideContainer, {
+      pointerId: 5,
+      buttons: 1,
+      clientX: 410,
+    });
+  });
+
+  const thumbs = container.querySelectorAll('.iui-slider-thumb');
+  expect(thumbs.length).toBe(2);
+
+  expect(thumbs[0].getAttribute('aria-valuenow')).toEqual('40');
+  expect(thumbs[1].getAttribute('aria-valuenow')).toEqual('80');
+});
+
+it('should activate thumb on pointerDown and move to closest step on move/ no update handler', () => {
```

Comment:
looks same as above.. Sure there is an advantage of having 2 same tests with passed and not passed on change handlers?

Reply:
I was trying to get to as close to 100% coverage as I could.  Different part of code is exercised if there is no update set. If this is not a concern for you I will remove this test.

Reply:
hmm.. Makes sense to test both cases, I just wonder if we could somehow merge some tests to test that instead of having a new one only for that (seems just too much of the code for a simple thing..)